main.go

```diff
@@ -198,6 +199,9 @@ func InitFlags(fs *pflag.FlagSet) {
 	fs.IntVar(&restConfigBurst, "kube-api-burst", 30,
 		"Maximum number of queries that should be allowed in one burst from the controller client to the Kubernetes API server. Default 30")
 
+	fs.DurationVar(&drainKubeTimeout, "drain-api-timeout", time.Second*10,
```

Comment:
```suggestion
	fs.DurationVar(&nodeDrainClientTimeout, "node-drain-client-timeout-duration", time.Second*10,
```

(added the suffix to hint at that the value is a duration to be consistent with the KCP controller flags)